summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -196,9 +216,31 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
       StormStatProvider.registerMetrics(jobID, context, countersForSpout)
       SummingbirdRuntimeStats.addPlatformStatProvider(StormStatProvider)
     })
-    val stormSpout = tormentaSpout.openHook(registerAllMetrics.get).getSpout
-    val parallelism = getOrElse(stormDag, node, parOpt.getOrElse(DEFAULT_SOURCE_PARALLELISM)).parHint
-    topologyBuilder.setSpout(nodeName, stormSpout, parallelism)
+    tormentaSpout = tormentaSpout.openHook(registerAllMetrics.get)
+
+    val summerOpt: Option[SummerNode[Storm]] = stormDag.dependantsOf(node.asInstanceOf[StormNode]).collect { case s: SummerNode[Storm] => s }.headOption
```

Comment:
actually, we do use it below, but we assume all summers downstream are batched the same. Maybe this is a bug elsewhere, but they don't have to be batched the same do they?